# Copyright (c) 2024. Tudor Oancea, Matteo Berthet
std_msgs/Header header
float64[] s_ref
float64[] n_ref
float64[] psi_ref
float64[] v_ref
float64[] delta_ref
float64[] tau_ref
float64[] s_pred
float64[] n_pred
float64[] psi_pred
float64[] v_pred
float64[] delta_pred
float64[] tau_pred
